settings.yml

```diff
@@ -5,3 +5,15 @@ host_url:
 website_id:
   type: string
   default: ''
+exclude_groups:
+  type: string
+  default: ''
+  description: Comma separated list of groups to exclude from tracking such as admins, staff.
```

Comment:
This is odd - typically we put the descriptions in `server.en.yml` - I see the `host_url` already did it this way, but I imagine it doesn't work in the app. Did you test in a browser to see if the description showed up in the admin section?

Reply:
I'm not familiar with Discourse code conventions but yes, it works. The descriptions show up in the theme components settings just like host_url. If there's a preferred place to put them I might need you to show me how. There isn't a server.en.yml in this component.

Reply:
It looks like it's my lack of knowledge with the theme component settings then! Apologies.